common/logger.go

```diff
@@ -47,6 +48,7 @@ type ILoggerCloser interface {
 type ILoggerResetable interface {
 	OpenLog()
 	MinimumLogLevel() pipeline.LogLevel
+	ChangeLogLevel(pipeline.LogLevel)
```

Comment:
`ILoggerResetable` inherits `ILoggerCloser` and `ILoggerCloser` inherits `ILogger`.
I see you are declaring `ChangeLogLevel()` in ILogger. So, by default `ChangeLogLevel()` method should be inherited by `ILoggerResetable`. Why do we need to declare it again?

Reply:
The cleaner solution would have been to create a new logger (at job manager level) to record job manager cleanup. This "cleanup logger" would inherit ILogger and could have an additional method `ChangeLogLevel()`. Doing this would not affect the existing code and keep the changes minimal. Is it feasible to do it based on your code changes?

Reply:
@nakulkar-msft @zezha-msft ?